csharp/ql/src/semmle/code/csharp/ir/implementation/ValueCategory.qll

```diff
@@ -0,0 +1,21 @@
+import csharp
```

Comment:
I don't think this file is even used anywhere. You should be able to just delete it. I'll delete it on the C++ side.